common/service/pldm/pldm_oem.h

```diff
@@ -91,7 +91,14 @@ enum oem_event_type {
 	CXL1_HB,
 	CXL2_HB,
 	POST_STARTED,
-	POST_ENDED
+	POST_ENDED,
```

Comment:
This change has conflicts. It needs to be rebased on top of this: https://github.com/facebook/OpenBIC/blob/main/common/service/pldm/pldm_oem.h#L95